code/online_challenges/src/project_euler/problem_040/problem_040.py

```diff
@@ -0,0 +1,18 @@
+# Made by adeen-s
+# Part of cosmos by OpenGenus
```

Comment:
Not needed.

Reply:
Well that's just an old habit from the hacktoberfest days. Fixed it all now.